traffic_report.rb

```diff
@@ -12,7 +12,7 @@ def visits_per_hour_past_day
     formatted = []
     rows.each do |row|
       formatted << {
-        hour: row[:dimension_values][0][:value],
+        hour: format_hour_value(row[:dimension_values][0][:value]),
         visits: row[:metric_values][0][:value]
```

Comment:
Can we use `dig` here?

Reply:
I changed it to `hour: format_hour_value(row.dig(:dimension_values).first.dig(:value))` but its crushed into the very last commit for formatting errors